packages/flutterfire_ui/lib/src/i10n/lang/hi.dart

```diff
@@ -0,0 +1,263 @@
+  @override
+  final String sendLinkButtonLabel;
+  @override
+  final String arrayLabel;
+  @override
+  final String booleanLabel;
+  @override
+  final String mapLabel;
+  @override
+  final String nullLabel;
+  @override
+  final String numberLabel;
+  @override
+  final String stringLabel;
+  @override
+  final String typeLabel;
+  @override
+  final String valueLabel;
+  @override
+  final String cancelLabel;
+  @override
+  final String updateLabel;
+  @override
+  final String northInitialLabel;
+  @override
+  final String southInitialLabel;
+  @override
+  final String westInitialLabel;
+  @override
+  final String eastInitialLabel;
+  @override
+  final String timestampLabel;
+  @override
+  final String latitudeLabel;
+  @override
+  final String longitudeLabel;
+  @override
+  final String geopointLabel;
+  @override
+  final String referenceLabel;
+
+  const EnLocalizations({
+    this.emailInputLabel = 'Email',
+    this.passwordInputLabel = 'Password',
+    this.signInActionText = 'Sign in',
+    this.registerActionText = 'Register',
+    this.signInButtonText = 'Sign in',
+    this.registerButtonText = 'Register',
+    this.linkEmailButtonText = 'Agla',
+    this.signInWithPhoneButtonText = 'Sign in with phone',
+    this.signInWithGoogleButtonText = 'Sign in with Google',
+    this.signInWithAppleButtonText = 'Sign in with Apple',
+    this.signInWithTwitterButtonText = 'Sign in with Twitter',
+    this.signInWithFacebookButtonText = 'Sign in with Facebook',
+    this.phoneVerificationViewTitleText = 'Apna phone number daalein',
+    this.verifyPhoneNumberButtonText = 'Next',
+    this.verifyCodeButtonText = 'Verify',
+    this.verifyingPhoneNumberViewTitle = 'Enter code from SMS',
+    this.unknownError = 'An unknown error occurred',
+    this.smsAutoresolutionFailedError =
+        'Failed to resolve SMS code automatically. Manually code enter karein',
+    this.smsCodeSentText = 'SMS code sent',
+    this.sendingSMSCodeText = 'Sending SMS code...',
+    this.verifyingSMSCodeText = 'Verifying SMS code...',
+    this.enterSMSCodeText = 'Enter SMS code',
+    this.emailIsRequiredErrorText = 'Email is required',
+    this.isNotAValidEmailErrorText = 'Email provide karein',
+    this.userNotFoundErrorText = "Account doesn't exist",
+    this.emailTakenErrorText = 'Account with such email already exists',
+    this.accessDisabledErrorText =
+        'Access to this account has been temporarily disabled',
+    this.wrongOrNoPasswordErrorText =
+        'The password is invalid or the user does not have a password',
+    this.signInText = 'Sign in',
+    this.registerText = 'Register',
+    this.registerHintText = "Don't have an account?",
+    this.signInHintText = 'Already have an account?',
+    this.signOutButtonText = 'Sign out',
+    this.phoneInputLabel = 'Phone number',
+    this.phoneNumberInvalidErrorText = 'Phone number is invalid',
+    this.phoneNumberIsRequiredErrorText = 'Phone number zaroori hai',

```

Comment:
Use proper language script all over your translations. Currently, they are Hindi-In-English (Not proper Hindi translations)